docs/content/logging-and-monitoring/service-insight.md

```diff
@@ -25,7 +25,7 @@ The service is healthy if at least one upstream is healthy. In this case, the en
 ## Enabling Service Insight Endpoint
 
 If you're using *Kubernetes manifests* (Deployment or DaemonSet) to install the Ingress Controller, to enable the Service Insight endpoint:
-1. Run the Ingress Controller with the `-enable-service-insight` [command-line argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments). This will expose the Ingress Controller endpoint via the path `/probe/{hostname}` on port `9114` (customizable with the `-service-insight-listen-port` command-line argument).
+1. Run the Ingress Controller with the `-enable-service-insight` [command-line argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments). This will expose the Ingress Controller endpoint via paths `/probe/{hostname}` for Virtual Servers, and `/probe/ts/{name}` for Transport Servers on port `9114` (customizable with the `-service-insight-listen-port` command-line argument).
```

Comment:
Probably need more clarification here on what `{name}` is and how is it different from `{hostname}`

Reply:
I'm inclined to agree - based on another part of this PR (I'm still learning k8s!) I'm under the impression they're somewhat synonymous - `hostname` is used for Virtual Servers and `name` is used for Transport Servers. Clarification on the nouns might not be necessary if this is a common convention.

Reply:
Updated